### PR TITLE
Improve Documentation (issue 2989)

### PR DESCRIPTION
**Pull request by spaetz**
This is a smallish attempt to improve openshot's sparse documentation
(issue #2989).
The getting started page only contains an installation description, so name
it so. Improve the quick tutorial somewhat (at least according to my taste)
but still keep it quick. Improve the start of the "Files" pages somewhat.

Make the import files table more consistent by nameing the method in the
first column and actually describing them in a full senctence (interpunction)
in the 2nd column. Add the actual toolbar button icon image to the table.

Far more is needed, but this is my first take at something easy.

### CODE REVIEW

**Comment by Ravi-Mangaluru** [comment]
Dear Team OpenShot,
Kindly provide the PDf format of the complete user guide for users to print and use them as Reference while working.

**Comment by spaetz** [comment]
Not sure why issue 2989 does not show this pull request as related pull request...

**Comment by ferdnyc** [comment]
@spaetz 

Thanks! I'm ashamed to say I haven't had a chance to look at this yet.

> Not sure why issue 2989 does not show this pull request as related pull request...

To relate PRs to Issues, you need to use one of Github's [keywords](https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword):

> - close
> - closes
> - closed
> - fix
> - fixes
> - fixed
> - resolve
> - resolves
> - resolved

However, as the names imply, adding that kind of linking means that the related issue will auto-close when the PR is merged, which I'm not sure is the desired effect on a non-specific issue regarding something as open-ended as documentation.

(There's no way to set up a "non-closing relation" unfortunately, submitters sometimes add notes that a PR "addresses #FOO" or is "related to #BAR" as an intentional way of relating them without setting up an auto-close, since it at least does get a "mentioned" note placed in the issue — as your initial description did, on this one.)

**Comment by spaetz** [comment]
@ferdnyc no need to apologize,  no hurry there. As long as you merge it now, of course ;-).
I use OpenShot on a near daily basis to create university video lectures and am happy to reciprocate.

File: doc/clips.rst [hunk 1]
```diff
@@ -43,25 +43,33 @@ Overview
...
    ==  ==================  ============
 
+.. _clips_cutting_slicing_ref:
+
 Cutting & Slicing
 -----------------
 OpenShot has many easy ways to adjust the start and end positions of a clip (otherwise known as cutting). The most common
 method is simply grabbing the left (or right) edge of the clip and dragging. Here is a list of methods for cutting clips in OpenShot:
 
+.. |razortool_icon| image:: ../src/images/Humanity/actions/16/edit-cut.svg
```
Comment (ferdnyc) [inline doc/clips.rst:53]:
```suggestion
.. |razortool_icon| image:: ../images/Humanity/actions/16/edit-cut.svg
```
The file location has changed in the repo.

File: doc/files.rst [hunk 2]
```diff
@@ -20,27 +20,42 @@
...
+
+
+.. _import_files_ref:
 
 Import Files
 ------------
-There are many ways to import media files into OpenShot.
+These are all possible methods to import media files into OpenShot:
+
+.. |importfiles_icon| image:: ../src/images/Humanity/actions/16/list-add.svg
```
Comment (ferdnyc) [inline doc/files.rst:39]:
```suggestion
.. |importfiles_icon| image:: ../images/Humanity/actions/16/list-add.svg
```

File: doc/installation.rst [hunk 3]
```diff
@@ -0,0 +1,56 @@
+.. Copyright (c) 2008-2016 OpenShot Studios, LLC
```
Comment (ferdnyc) [inline doc/installation.rst:1]:
```suggestion
.. Copyright (c) 2008-2020 OpenShot Studios, LLC
```
Might as well be current for NEW files. :wink:

File: doc/quick_tutorial.rst [hunk 4]
```diff
@@ -26,35 +26,45 @@ Using OpenShot is very easy, and this tutorial will take you through the basics
...
-------------------------------
-Before we can begin making a video, we need to import files into OpenShot. Drag and
-drop a few images (\*.JPG, \*.PNG, etc...) and a music file (most formats will work)
-from your Desktop to OpenShot. Be sure to drop the files where the
+Step 1 – Import media files
+---------------------------
+
+Before we can begin making a video, we need to import media files into OpenShot. Most video,
+image and music file formats will work. Drag and drop a few videos or images and a music file
+(most formats will work) from your Desktop to OpenShot. Be sure to drop the files where the
```
Comment (ferdnyc) [inline doc/quick_tutorial.rst:34]: This is a bit redundant in terms of "most formats will work", saying it twice in two sentences.

File: doc/quick_tutorial.rst [hunk 5]
```diff
@@ -63,7 +73,8 @@ buttons.
 
 Step 5 – Export your Video
 ---------------------------
-Once you are happy with your photo slide-show video, the next step is to export your video.
+
+Once you are happy with your video, the next step is to export your video.
```
Comment (ferdnyc) [inline doc/quick_tutorial.rst:77]: Ditto here, re: redundancy. Perhaps "Once you are happy with your layout...", or "Once you are happy with your project..."?

**Review by ferdnyc — CHANGES_REQUESTED** [review]
@spaetz Finally got a chance to review this, thanks for submitting! (Sorry about the wait, again.) I noted a couple of changes needed (Primarily the image paths, other stuff is at-your-discretion really.)

Also, it's probably a good idea to edit the `index.rst` file, to insert the `installation` page into the TOC wherever it should go. (I don't have an immediate answer to that question myself, but point is it's good to tell Sphinx explicitly what order to place the sections in.)

**Comment by spaetz** [comment]
> Also, it's probably a good idea to edit the index.rst file, to insert the installation page into the TOC wherever it should go.

Pfeww, glad this is in my MR then:
```
-   getting_started
+   installation
```
Will fix the other suggestions, thanks for reviewing.

File: doc/files.rst (hunk 2 above)
Comment (spaetz) [reply doc/files.rst:39]: Thanks, done

File: doc/installation.rst (hunk 3 above)
Comment (spaetz) [reply doc/installation.rst:1]: It is merely renamed to be pendantic :). But yes, updated.

File: doc/quick_tutorial.rst (hunk 4 above)
Comment (spaetz) [reply doc/quick_tutorial.rst:34]: Copy and paste leftover, fully agreed. Gone.

File: doc/clips.rst (hunk 1 above)
Comment (ferdnyc) [reply doc/clips.rst:53]: This change got missed in the update, but I can apply it here from my suggestion.

File: doc/installation.rst (hunk 3 above)
Comment (ferdnyc) [reply doc/installation.rst]: Ah, I didn't actually realize that. Still, new name new year, why not? :laughing:

**Review by ferdnyc — APPROVED** [review]

**Comment by ferdnyc** [comment]
Once this get through Travis, I'll merge. Many thanks, @spaetz !

**Pull request merged**